src/plugins/data/public/query/query_string/dataset_service/types.ts

```diff
@@ -16,6 +16,14 @@ export interface DataStructureFetchOptions {
   paginationToken?: string;
 }
 
+export interface IndexedView {
```

Comment:
Please prefix this and everything else that exported from here with `Dataset`

Reply:
addressed in https://github.com/opensearch-project/OpenSearch-Dashboards/pull/8818